### ISSUE TITLE
KeyError when using SparseEnum sparsely with default()

### ISSUE THREAD

**Issue by jonemo** [TadLeonard/enumap#4]
Using the `default()` feature introduced in version `1.3.0` (via #1) renders breaks `SparseEnumap`, omitted keys trigger a `KeyError`:

```
In [1]: from enumap import SparseEnumap, default

In [2]: from enum import auto

In [4]: class Box(SparseEnumap):
   ...:     a: str = auto()
   ...:     b: str = default('b')
   ...:

In [5]: Box.tuple(b='bb')
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/Users/jonas/.virtualenvs/cadsift/lib/python3.6/site-packages/enumap.py in tuple(cls, *values, **named_values)
     61         try:
---> 62             return tuple_class(*values, **named_values)
     63         except TypeError:

TypeError: __new__() missing 1 required positional argument: 'a'

During handling of the above exception, another exception occurred:

KeyError                                  Traceback (most recent call last)
<ipython-input-5-ff78554e8514> in <module>()
----> 1 Box.tuple(b='bb')

/Users/jonas/.virtualenvs/cadsift/lib/python3.6/site-packages/enumap.py in tuple(cls, *values, **named_values)
     62             return tuple_class(*values, **named_values)
     63         except TypeError:
---> 64             mapping = cls._make_checked_mapping(*values, **named_values)
     65             return tuple_class(**mapping)
     66

/Users/jonas/.virtualenvs/cadsift/lib/python3.6/site-packages/enumap.py in _make_checked_mapping(cls, *values, **named_values)
    236             mapping = dict(zip(names, values), **named_values)
    237             missing = names_set - set(mapping)
--> 238             mapping.update(((k, defaults[k]) for k in missing))
    239         else:
    240             mapping = dict(zip_longest(names, values), **named_values)

/Users/jonas/.virtualenvs/cadsift/lib/python3.6/site-packages/enumap.py in <genexpr>(.0)
    236             mapping = dict(zip(names, values), **named_values)
    237             missing = names_set - set(mapping)
--> 238             mapping.update(((k, defaults[k]) for k in missing))
    239         else:
    240             mapping = dict(zip_longest(names, values), **named_values)

KeyError: 'a'
```

When not using `default()`, the `SparseEnumap` behaves as expected and as before:

```
In [6]: class Box(SparseEnumap):
   ...:     a: str = auto()
   ...:     b: str = auto()
   ...:

In [7]: Box.tuple(b='bb')
Out[7]: Box_tuple(a=None, b='bb')
```

**Comment by jonemo** [comment]
Solution idea: Should https://github.com/TadLeonard/enumap/blob/master/enumap.py#L219 be

```diff
-             member_defaults = dict(_iter_member_defaults(members))
+             member_defaults = cls.map(**dict(_iter_member_defaults(members)))
```

**Comment by TadLeonard** [comment]
Thank you! I'll push out a `1.3.1` ASAP.

**Comment by TadLeonard** [comment]
Fixed by #5
